content/en/docs/overview/score-for-devs.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Score for developers"
+linkTitle: "Score for developers"
+description: Learn about Score as a developer
+weight: 1
+Focus_Keyword: "Learn about Score as a developer"
+
+---
+
+## Experience Score as a Developer
+
+Score is a workload specification designed to simplify development for cloud-native developers. The spec enables you to describe your workload’s configuration in a vendor-neutral way, eliminating the need for tooling-specific syntax from platforms such as Docker or Kubernetes for example. By leveraging familiar concepts and semantics, defining a workload’s configuration becomes as simple as stating, "I want a database of type X and an event queue of type Y to accompany my workload".
+
+Below you'll find an example of a simple Score application with a web server that queries a Postgres database on each request. The demo code can be found [here](https://github.com/score-spec/sample-app-gif/tree/d3783c25aba1a89db37659a355df11fd42f8da10).
```

Comment:
Any particular reason we are targeting a specific commit? Otherwise I suggest we target `main` so that updates to the repo are reflected for anyone following the link.

```suggestion
Below you'll find an example of a simple Score application with a web server that queries a Postgres database on each request. The demo code can be found [here](https://github.com/score-spec/sample-app-gif).
```